oam/install_scripts/module_installer.sh

```diff
@@ -134,43 +124,45 @@ MySQLRep=`$COLUMNSTORE_INSTALL_DIR/bin/getConfig Installation MySQLRep`
 if [ $MySQLRep = "y" ]; then
 	if test -f $COLUMNSTORE_INSTALL_DIR/mysql/my.cnf ; then
 		echo "Run Upgrade on my.cnf on Module"
-		$COLUMNSTORE_INSTALL_DIR/bin/mycnfUpgrade > /tmp/mycnfUpgrade.log 2>&1
+		$COLUMNSTORE_INSTALL_DIR/bin/mycnfUpgrade > ${tmpDir}/mycnfUpgrade.log 2>&1
 	fi
 fi
 
 if test -f $COLUMNSTORE_INSTALL_DIR/mysql/my.cnf ; then
 	mysqlPort=`$COLUMNSTORE_INSTALL_DIR/bin/getConfig Installation MySQLPort`
 	echo "Run Mysql Port update on my.cnf on Module"
-	$COLUMNSTORE_INSTALL_DIR/bin/mycnfUpgrade $mysqlPort > /tmp/mycnfUpgrade_port.log 2>&1
+	$COLUMNSTORE_INSTALL_DIR/bin/mycnfUpgrade $mysqlPort > ${tmpDir}/mycnfUpgrade_port.log 2>&1
 fi
 
 # if um, run mysql install scripts
 if [ $module = "um" ] || ( [ $module = "pm" ] && [ $PMwithUM = "y" ] ) || [ $ServerTypeInstall = "2" ]; then
 	echo "Run post-mysqld-install"
-	$COLUMNSTORE_INSTALL_DIR/bin/post-mysqld-install --installdir=$COLUMNSTORE_INSTALL_DIR > /tmp/post-mysqld-install.log 2>&1
+	$COLUMNSTORE_INSTALL_DIR/bin/post-mysqld-install --installdir=$COLUMNSTORE_INSTALL_DIR > ${tmpDir}/post-mysqld-install.log 2>&1
 	if [ $? -ne 0 ]; then
-	    echo "ERROR: post-mysqld-install failed: check /tmp/post-mysqld-install.log"
+	    echo "ERROR: post-mysqld-install failed: check ${tmpDir}/post-mysqld-install.log"
 	    exit 1
 	fi
 	echo "Run post-mysql-install"
 	
-	$COLUMNSTORE_INSTALL_DIR/bin/post-mysql-install --installdir=$COLUMNSTORE_INSTALL_DIR  > /tmp/post-mysql-install.log 2>&1
+	$COLUMNSTORE_INSTALL_DIR/bin/post-mysql-install --installdir=$COLUMNSTORE_INSTALL_DIR  --tmpdir=${tmpDir} > ${tmpDir}/post-mysql-install.log 2>&1
         if [ $? -ne 0 ]; then
-            echo "ERROR: post-mysql-install failed: check /tmp/post-mysql-install.log"
+            echo "ERROR: post-mysql-install failed: check ${tmpDir}/post-mysql-install.log"
             exit 1
 	fi
 fi
 
-$COLUMNSTORE_INSTALL_DIR/bin/syslogSetup.sh check > /tmp/syslogSetup-check.log 2>&1
-if [ $? -ne 0 ]; then
-	# try setup again
-	$COLUMNSTORE_INSTALL_DIR/bin/syslogSetup.sh install > /tmp/syslogSetup-install.log 2>&1
-	if [ $? -ne 0 ]; then
-		echo "WARNING: syslogSetup.sh check failed: check /tmp/syslogSetup-check.log"
-       		exit 2
-	fi
+if [ $user == "root" ]; then
+    $COLUMNSTORE_INSTALL_DIR/bin/syslogSetup.sh check > ${tmpDir}/syslogSetup-check.log 2>&1
+    if [ $? -ne 0 ]; then
+	    # try setup again
+	    $COLUMNSTORE_INSTALL_DIR/bin/syslogSetup.sh install > ${tmpDir}/syslogSetup-install.log 2>&1
+	    if [ $? -ne 0 ]; then
+		    echo "WARNING: syslogSetup.sh check failed: check ${tmpDir}/syslogSetup-check.log"
+		    exit 2
+	    fi
+    fi
 fi
```

Comment:
It's good that we're checking the user here since this messes with the syslog configuration.  We might need a clause for when the user isn't root, maybe a printout to tell the user .. what .. ?  A couple options 1) that syslogsetup should be run as root, or 2) where the logs will go instead...?